src/lib/utils/path.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
See https://github.com/GoogleChrome/web.dev/pull/9413#pullrequestreview-1263713377 for details.